README.md

```diff
@@ -20,8 +20,8 @@ Once a Mist version is released the Meteor frontend part is bundled using `meteo
 
 To run mist in development you need [Node.js NPM](https://nodejs.org) and [Meteor](https://www.meteor.com/install) installed. When this is done, install Electron:
 
-    $ npm install -g electron-prebuilt
     $ curl https://install.meteor.com/ | sh
+    $ sudo npm install -g electron-prebuilt
```

Comment:
If you use sudo than you configured your mac wrong ;)
http://stackoverflow.com/questions/16151018/npm-throws-error-without-sudo